pandas/core/generic.py

```diff
@@ -10897,6 +10900,12 @@ def mad(
         {see_also}\
         {examples}
         """
+        msg = (
+            "The 'mad' method is deprecated "
+            "and will be removed in a future version. "
```

Comment:
I think it would be good to say here how to calculate mad with existing functionality to help users migrate.

Reply:
Thanks @mroeschke - added.